### PR TITLE
Added an option of updating children packages

### PR DESCRIPTION
**Pull request by quesabe**
Closes PLA-75.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +21,29 @@ steps:
...
+```yaml
+steps:
+  - uses: ottofeller/create-release-gh-action@main
+  with:
+    initial-version: 0.0.1
+    release-branches: main,master,dev
+    bump-level: minor
+    github-token: ${{ secrets.GITHUB_TOKEN }}
+    update-main-package_json: true
+    update-children-path: packages/first packages/second packages/third
```
Comment (gvidon) [inline README.md:34]: @quesabe can we have the list of repos own the shape of yaml array? Like this:
```yaml
update-children-path:
  - package/first
  - package/second
  - package/third

# OR

update-children-path: ['package/first', 'package/second', 'package/third']
```

File: README.md [hunk 2]
```diff
@@ -20,13 +21,29 @@ steps:
...
+**Advanced:**
+```yaml
+steps:
+  - uses: ottofeller/create-release-gh-action@main
+  with:
+    initial-version: 0.0.1
+    release-branches: main,master,dev
+    bump-level: minor
+    github-token: ${{ secrets.GITHUB_TOKEN }}
+    update-main-package_json: true
```
Comment (gvidon) [inline README.md:33]: @quesabe please rename to this value (does this flag require the update of the root `packages.json`, and if not provided will only update nested `package.json`s):
```suggestion
    update-root-package-json: true
```

File: README.md (hunk 1 above)
Comment (gvidon) [reply README.md:34]: Please don't forget to update the docs accordingly.

**Review by gvidon — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (quesabe) [reply README.md]: Unfortunately actions take only strings as inputs. However the list might be separated with any spaces, including new lines. Thus a multiline string is a comparable option. I have included the syntax in an example in README.

Comment (gvidon) [reply README.md]: @quesabe ok, got it, let's keep it like this.

**Pull request merged**